javatests/dagger/android/support/functional/TestActivity.java

```diff
@@ -33,6 +33,7 @@ protected void onCreate(Bundle savedInstanceState) {
     getSupportFragmentManager()
         .beginTransaction()
         .add(new TestParentFragment(), "parent-fragment")
+        //.add(new TestDialogFragment(), "dialog-fragment")
```

Comment:
Did you mean to uncomment this?

Reply:
Yep, I wasn't sure if this is the best place to put it though, or whether it should go in `TestChildFragment` for example, what do you think?

Reply:
Either would probably work right? A dialog fragment doesn't have to be a child of another fragment IIRC. This seems fine to me

Reply:
Yep it's fine for it be a sibling too, cool I'll uncomment